docs/standard/garbage-collection/fundamentals.md

```diff
@@ -86,7 +86,7 @@ manager: "wpickett"
 ## Conditions for a garbage collection  
  Garbage collection occurs when one of the following conditions is true:  
   
--   The system has low physical memory.  
+-   The system has low physical memory, such as when the system receives a low memory notification from the operating system.  
```

Comment:
I don't think the "system" here is meant to be .Net, it's meant to be the machine/operating system. If mentioning notifications makes sense here, i would write it like this:

> The system has low physical memory. The runtime can learn about this by receiving a low memory notification from the operating system.

Reply:
you can say "This is detected by either the low memory notification from the OS or low memory indicated by the host"